gitgud/skills/user_messages.py

```diff
@@ -74,3 +74,15 @@ def all_levels_complete():
 def default_fail():
     print('Level not complete, keep trying. "git gud reset" to start from scratch.')
 
+@separated
```

Comment:
Is this something that needs to be separated? This decorator separates two blocks of text with one empty line. Are we ever using this with multiple blocks of texts (a block of text is equal to one other method with the `@separated` decorator)